Modules/ScormAicc/scripts/basisAPI.js

```diff
@@ -337,7 +358,21 @@ function IliasCommit() {
 		if (typeof SOP!="undefined" && SOP==true) ret=saveRequest(o_data);
 		else {
 			s_s=toJSONString(o_data);
-			ret=sendRequest ("./storeScorm.php?package_id="+iv.objId+"&ref_id="+iv.refId+"&client_id="+iv.clientId+"&do=store", s_s);
+			if (isChromium80plus() && isBeforeUnloadEvent()) {
+				// ATTENTION: This is a mean hack!
+				//
+				// With Chrome 80+ it became deprecated to send async requests
+				// during beforeunload:
+				// 	- https://www.chromestatus.com/feature/4664843055398912
+				// 	- https://mantis.ilias.de/view.php?id=28363
+				// This is why we send and async request here and just pretend
+				// that everything was "ok".
+				sendRequest ("./storeScorm.php?package_id="+iv.objId+"&ref_id="+iv.refId+"&client_id="+iv.clientId+"&do=store", s_s, function () {});
+				ret="ok";
```

Comment:
Das bedeutet, wenn etwas schiefgehen würde, würde man das nicht mitbekommen? Da wird man m.E. nichts anderes machen können, oder?

Reply:
As far as I understand the state of affairs, the SCORM-API is synchronous here, so an immediate answer is expected by the client. I cannot see how we might get that answer with an async request. So yes, the client won't notice if something went wrong. But: since we are just in the process of closing the browser there is not much he could do anyway, even if we would detect a problem. Also: the current situation for Chrome 80+ is complete loss of data, this should at least be better =)

Reply:
absolutely, everything is better than the current situation :-)